Library/Homebrew/cmd/style.rb

```diff
@@ -1,3 +1,18 @@
+#:  * `style` [`--fix`] <targets>
```

Comment:
Two issues:

1. This line needs to end in a colon (compare with other commands). Otherwise the styling as a definition list is broken and this is causing the huge diff in `brew.1.html` and `brew.1`.

2. `<targets>` are optional, i.e. make that `[<targets>]`. IMO, even better would be to make that `[<formulae>|<paths>]` and to adjust the text below accordingly.